flink-runtime/src/test/java/org/apache/flink/runtime/security/modules/JaasModuleTest.java

```diff
@@ -59,10 +63,21 @@ public void testJaasModuleFilePathIfWorkingDirNotPresent() throws IOException {
 		testJaasModuleFilePath(file.toPath().toString() + "/tmp");
 	}
 
+	@Test
+	public void testJaasModuleFilePathIfWorkingDirIsSymLink() throws IOException {
+		File baseFolder = folder.newFolder();
+		File actualFolder = new File(baseFolder, "actual_folder");
+		assertTrue(actualFolder.mkdir());
+
+		Path symlink = new File(baseFolder, "symlink").toPath();
+		Files.createSymbolicLink(symlink, actualFolder.toPath());
```

Comment:
We may wrap that logic to something like `createTempFolderAndItsSymLink`. Then, we could test the symbolic link case with `testJaasModuleFilePath(symlink.toString() + "/tmp")`. WDYT?